fdbserver/include/fdbserver/IPageEncryptionKeyProvider.actor.h

```diff
@@ -339,7 +339,7 @@ class TenantAwareEncryptionKeyProvider : public IPageEncryptionKeyProvider {
 
 	std::tuple<int64_t, size_t> getEncryptionDomain(const KeyRef& key, Optional<int64_t> possibleDomainId) override {
 		// System key.
-		if (key.startsWith("\xff\xff"_sr)) {
+		if (key.startsWith("\xff"_sr)) {
```

Comment:
There is a `isSystemKey` method you can use

Reply:
https://github.com/apple/foundationdb/blob/main/fdbserver/include/fdbserver/ApplyMetadataMutation.h#L108

Reply:
+1.